CMPUT301F24bubble/bubbletracks
# Profile pictures

testing
- go to profile, your profile picture should be a deterministically generated one now. 
- change your name. it should also change the automatically generated photo. 
- upload a profile picture. the change should be showed in firebase and in profile screen.
- go to admin browse profiles, you should be able to delete a profile picture now.
